lms/views/lti/deep_linking.py

```diff
@@ -136,7 +136,8 @@ def file_picker_to_form_fields_v13(self):
         #
         # This claim is required if present in the `LtiDeepLinkingRequest`
         # message.
-        if data := self.request.parsed_params["deep_linking_settings"].get("data"):
+        settings = self.request.parsed_params["deep_linking_settings"] or {}
```

Comment:
`deep_linking_settings` is present but the value is None in the exceptions in sentry.

Reply:
Another option here is a double get play:

```python
self.request.parsed_params.get("deep_linking_settings", {}).get("data")
```

Reply:
I tried that first but I always forget `get` doesn't do a falsy test, just presence. So `{"deep_linking_settings":  None}.get` fails.